doc/nrf/device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield.rst

```diff
@@ -0,0 +1,88 @@
+.. _peripheral_sensor_node_shield:
+
+Peripheral Sensor node Shield
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The Peripheral Sensor node shield is an extension for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`.
+It is a testing shield with following sensors:
+
+   * ``ADXL362`` - Accelerometer (SPI interface)
+   * ``BMI270`` - Accelerometer and gyroscope (SPI interface)
+   * ``BME688`` - Temperature, pressure, humidity and gas sensor (I2C interface with on-board pull-up resistors 4.7 kohm).
+                  It is targeted for testing I2C, and SPI protocols drivers.
+
+Supply voltage - 1.8V
+
+.. figure:: /images/peripheral_sensor_node_shield.png
+   :width: 350px
+   :align: center
+   :alt: Peripheral Sensor node shield
+
+   Peripheral Sensor node shield
+
+Pin assignment
+**************
+
+For the exact pin assignment, refer to the following table:
+
++-------------------------+------------+-----------------+
+| nRF54L15 connector pin  | SIGNAL     | Shield function |
++=========================+============+=================+
+| P1.13                   | BMI270 CS  | Chip Select     |
++-------------------------+------------+-----------------+
+| P2.10                   | ADXL362 CS | Chip Select     |
++-------------------------+------------+-----------------+
+| P2.08                   | SPI MOSI   | Serial Data In  |
++-------------------------+------------+-----------------+
+| P2.09                   | SPI MISO   | Serial Data Out |
++-------------------------+------------+-----------------+
+| P2.06                   | SPI SCK    | Serial Clock    |
++-------------------------+------------+-----------------+
+| P1.14                   | GPIO       | SPI Interrupt   |
++-------------------------+------------+-----------------+
+| P1.11                   | TWI SCL    | TWI Clock       |
++-------------------------+------------+-----------------+
+| P1.12                   | TWI SDA    | TWI Data        |
++-------------------------+------------+-----------------+
+
+Requirements
+************
+
```

Comment:
I wonder if the entire section is necessary. Would it make sense for the screenshot to go under the pin assignment and then just to mention in a note box for better visibility the following:
``The shield must be oriented in a way that allows its silkscreen to read the same way as PDK.
The shield must not be rotated.`` ?

Reply:
We can do that.